code/modules/surgery/organs/stomach.dm

```diff
@@ -162,3 +162,22 @@
 		COOLDOWN_START(src, severe_cooldown, 10 SECONDS)
 	if(prob(emp_vulnerability/severity))	//Chance of permanent effects
 		organ_flags |= ORGAN_SYNTHETIC_EMP //Starts organ faliure - gonna need replacing soon.
+
+/obj/item/organ/stomach/cell
+	name = "micro-cell"
+	icon_state = "microcell"
+	w_class = WEIGHT_CLASS_NORMAL
+	zone = "chest"
+	slot = "stomach"
+	attack_verb = list("assault and battery'd")
+	desc = "A micro-cell, for IPC use only. Do not swallow."
+	status = ORGAN_ROBOTIC
+
+/obj/item/organ/stomach/cell/emp_act(severity)
+	switch(severity)
+		if(1)
+			owner.nutrition = 50
+			to_chat(owner, "<span class='warning'>Alert: Heavy EMP Detected. Rebooting power cell to prevent damage.</span>")
+		if(2)
+			owner.nutrition = 250
```

Comment:
me and the boys EMPing ourselves while low on nutrition to set our nutrition to a higher value:
![image](https://media3.giphy.com/media/l1J9LDFOHRekAn9jW/source.gif)